### ISSUE TITLE
ComboBox component  DAP issue - aria-labelledby on element div is not valid

### ISSUE THREAD

**Issue by Kandianne** [carbon-design-system/carbon#5939]
## What package(s) are you using?

- [ ] `carbon-components`
- [x] `carbon-components-react`

## Detailed description

Getting DAP violation that "_The id combobox-label-4 specified for WAI-ARIA property aria-labelledby on element div is not valid. See rule help for remediation guidance._"

**Related to the ComboBox**

**Would like to see DAP violation fixed or know how to fix.**

**Viewing in Chrome version 80.0.3987.163**

**using carbon-components-react v7.11.3**

## Steps to reproduce the issue

1. Visit  code sandbox here https://codesandbox.io/s/heuristic-wescoff-73k57
2. Run DAP and see violations 
![image](https://user-images.githubusercontent.com/12722441/80258723-e921c980-8651-11ea-8656-da11f1aad46e.png)

**Comment by joshblack** [comment]
FYI @dakahn is this tackled by your PR?

**Comment by dakahn** [comment]
closed by https://github.com/carbon-design-system/carbon/pull/5373

**Comment by Kandianne** [comment]
Hi @dakahn will there be a release, including this fix, coming out soon?
